yt_dlp/extractor/afreecatv.py

```diff
@@ -238,7 +246,7 @@ def _entries(data):
                 'title': ('title', {str}),
                 'uploader': ('writer_nick', {str}),
                 'uploader_id': ('writer_id', {str}),
-                'thumbnail': ('thumb', {url_or_none}),
+                'thumbnails': ('thumb', {AfreecaTVBaseIE._fixup_thumb}),
```

Comment:
Consistency with above?
```suggestion
                'thumbnails': ('thumb', {self._fixup_thumb}),
```

Reply:
we are in a static method here so can't use `self`

Reply:
reason to not make it a classmethod then?